lib/AST/GenerateAST.cpp

```diff
@@ -590,7 +616,12 @@ GenerateAST::Result GenerateAST::run(llvm::Function &func,
   // structurization
   llvm::ReversePostOrderTraversal<llvm::Function *> rpo(&func);
   rpo_walk.assign(rpo.begin(), rpo.end());
-  GetOrCreateReachingCond(rpo_walk[0]);
+  do {
```

Comment:
Can you briefly describe in a comment why this fixpoint computation is needed?

Reply:
Done!